vxlanBridge.go

```diff
@@ -789,3 +808,48 @@ func (self *Vxlan) processArp(pkt protocol.Ethernet, inPort uint32) {
 		}
 	}
 }
+
+// sendGARP sends GARP for the specified IP, MAC
+func (self *Vxlan) sendGARP(ip net.IP, mac net.HardwareAddr, vni uint64) error {
+
+	// NOTE: This is disabled for now. Will be added when EVPN support is added.
+	if !VXLAN_GARP_SUPPORTED {
+		return nil
+	}
+
+	garpPkt, _ := protocol.NewARP(protocol.Type_Request)
```

Comment:
It'll be good to create a utility function to build GARP packet instead of duplicating it..